import/update.sh

```diff
@@ -5,33 +5,44 @@
 # This is free software, and you are welcome to redistribute it under certain conditions.
 # See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 
+set -e
 
 # extend environment paths by location of osmconvert, osmupdate and osmfilter
 export PATH=/usr/local/bin:/usr/local/sbin:${PATH}
 
 source $(dirname ${0})/config.cfg
 
+# default to current directory
+DATADIR=${DATADIR:-.}
+
 cd $PROJECTPATH/import
 
 echo "Started processing at $(date)"
 
 echo "[1/3] Fetching diff"
 date -u +%s > timestamp_tmp
 TIMESTAMP=$(<timestamp)
+TIMESTAMP_NEW=$(<timestamp_tmp)
 UPDATE=`date -u -d "@$TIMESTAMP" +%Y-%m-%dT%H:%M:%SZ`
-osmupdate $UPDATE changes.osc --day -v
-osmconvert changes.osc --drop-relations --out-osc > changes-norelation.osc
-rm changes.osc
+osmupdate $UPDATE ${DATADIR}/changes.osc -v
+
+if [ ! -s ${DATADIR}/changes.osc ]; then
+	echo "No new data available"
+	exit 0
+fi
+
+osmconvert ${DATADIR}/changes.osc --drop-relations --out-osc > ${DATADIR}/changes-norelation.osc
+rm ${DATADIR}/changes.osc
 
 echo "[2/3] Updating database"
 rm expired_tiles
```

Comment:
No, it's your turn to find out why this is actually there and if it should go away.